### PR TITLE
Major Description Changes

### PR DESCRIPTION
**Pull request by gonzalez-mark**
I don't think I did this right

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1499,7 +1499,7 @@ <h3>Esther Jung
                         <img src="img/team/jaimie.jpeg" class="committee-photo jaimie">
                         <figcaption class="caption text-center">
                             <h3>Jaimie Lee
-                                <small class="major">Public Health, Psychology, and Public Policy</small>
+                                <small class="major">Public Health, Psychology, & Public Policy</small>
```
Comment (KhanWhale) [inline index.html:1502]: Hm, I think there's a typo here (should just be a ",")

File: index.html [hunk 2]
```diff
@@ -1382,7 +1382,7 @@ <h3>Marie Bellevue
                         <figcaption class="caption text-center">
                             <h3>Aniruddh Khanwale
                                 <!-- <small>loves making google forms</small> -->
-                                <small class="major">Materials Science & Engineering, EECS</small>
+                                <small class="major">Materials Science and Engineering, EECS</small>
```
Comment (KhanWhale) [inline index.html]: There's nothing wrong here, but do you mind adding Energy Engineering at the end :O

File: index.html [hunk 3]
```diff
@@ -1793,22 +1793,22 @@ <h3>Matthew Clark</h3>
...
+        Hey there! I'm a physics major who also appreciates math (I know you didn't see that coming) and cares about the environment. When I'm not at the chalkboard, you can find me swing dancing, cooking, daydreaming about Siberian Huskies, exploring Berkeley for vegan treats, perpetually drinking tea, reading, and (most importantly!) sleeping. Home sweet home is back in Alabama, so if you have any concerns about being far from home, please share them with me! ROHP led to my first friendships at Cal, and I can't wait to share this Cal-tastic experience with you :) Please reach out if you'd like to talk about Physics, Math, EEP (Environmental & Economic Policy), succeeding in physics, getting research, going out-of-state for college, being vegan in Berkeley, deCals, 7-breadth requirements, Society of Physics Students, RCSA, ROHP, the benefits of being a Regents' Scholar, planning your first semester, on-campus housing, concerns about Berkeley.
```
Comment (KhanWhale) [inline index.html]: This doesn't look like it's your fault, but see if you can Shantanu's bio to be formatted like everyone else's (i.e. multiple lines). Same thing if there's any other bios that for some reason are only one line.

**Review by KhanWhale — CHANGES_REQUESTED** [review]
Do you mind removing committee.html **from the repo only?** Since this is still the "original" website, we don't have separate html files (yet). I don't want you to delete the file, because it will be useful on the redesign branch, just move it out of your "rohm-website" folder and make a new commit, also addressing the changes I mentioned. Overall though, looks really good, thank you!

File: index.html (hunk 1 above)
Comment (gonzalez-mark) [reply index.html:1502]: Since Jaimie has 3 majors listed, the structure is as follows: "major, major, & major".

Comment (KhanWhale) [reply index.html:1502]: Ah, I see.

**Review by KhanWhale — APPROVED** [review]
Looks good. Great job!

**Pull request merged**